src/controllers/remove.director.controller.ts

```diff
@@ -27,9 +37,47 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
   try {
     //Hardcoded for now
     req.params[urlParams.PARAM_SUBMISSION_ID] = "645d1188c794645afe15f5cc";
+
+    const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
+    const submissionId = "645d1188c794645afe15f5cc";
```

Comment:
Value could have been made into a constant for easy reuse

Reply:
This is from an old commit that is no longer in the code